sdk/identity/Azure.Identity/src/FileContentsCache.cs

```diff
@@ -18,9 +18,11 @@ internal class FileContentsCache
         private readonly string _tokenFilePath;
         private string _tokenFileContents;
         private DateTimeOffset _refreshOn = DateTimeOffset.MinValue;
-
-        public FileContentsCache(string tokenFilePath)
+        private readonly TimeSpan _refreshInterval;
```

Comment:
nit - add back the blank line between this and the ctor below

Reply:
Fixed.